path_creds.go

```diff
@@ -119,6 +124,34 @@ func (b *backend) pathCredentialsRead(ctx context.Context, req *logical.Request,
 	return b.createCreds(ctx, req, roleEntry, request)
 }
 
+func (b *backend) isValidKubernetesNamespace(ctx context.Context, req *logical.Request, namespace string, role *roleEntry) (bool, error) {
```

Comment:
This would be a good function to exercise with some unit tests (and a good excuse to finally add a path_creds_test.go).